src/garage/experiment/__init__.py

```diff
@@ -3,15 +3,11 @@
 from garage.experiment.experiment import to_local_command
 from garage.experiment.experiment import variant
 from garage.experiment.experiment import VariantGenerator
-from garage.experiment.snapshotter import Snapshotter
-
-# LocalRunner needs snapshotter to be imported, so we have to use a strange
-# import order here
-snapshotter = Snapshotter()
```

Comment:
this change is not related to the PR. did you introduce a merge commit, or branch from one of your other PRs?